src/api/config/routes.rb

```diff
@@ -366,13 +366,20 @@ def self.public_or_about_path?(request)
       get 'user/edit/:user' => :edit, constraints: cons, as: 'user_edit'
 
       get 'user/show/:user' => :show, constraints: cons, as: 'user_show'
+
       # Only here to make old /home url's work
       get 'home/' => :home, as: 'home'
       get 'home/my_work' => :home
       get 'home/list_my' => :home
       get 'home/home_project' => :home_project
     end
 
+    controller 'webui/users/announcements' do
```

Comment:
Is this the only way of doing it? I mean, can it be done with nested resources?

Reply:
Just gave it a try. The problem is that we need the announcement id (the user we get via the session). We would have to work around that and probably end up with a route which is somewhat different to all other routes we have in OBS.